src/ripple/protocol/impl/SOTemplate.cpp

```diff
@@ -21,42 +21,46 @@
 
 namespace ripple {
 
-void SOTemplate::push_back (SOElement const& r)
+SOTemplate::SOTemplate (std::initializer_list<SOElement> uniqueFields,
+    std::initializer_list<SOElement> commonFields)
+: elements_ ()
```

Comment:
Why the explicit default constructor call? I'd lose it. Also formatting seems off.

Reply:
I've removed the explicit constructor for `elements_`.  I'm curious about what seems off with the formatting.  If you can be specific I'll fix it.  Thanks.

Reply:
The member initializations (the `:` part) should be indented four spaces. So something like:
```

SOTemplate::SOTemplate(
    std::initializer_list<SOElement> uniqueFields,
    std::initializer_list<SOElement> commonFields)
    : indices_(SField::getNumFields() + 1, -1)  // Unmapped indices == -1
```